pulp_smash/tests/pulp3/pulpcore/api_v3/plugin_involved/test_repo_version.py

```diff
@@ -388,8 +388,9 @@ def test_filter_invalid_date(self):
                 {'created__gte': criteria, 'created__lte': criteria},
                 {'created__range': ','.join((criteria, criteria))}):
             with self.subTest(params=params):
-                versions = get_versions(self.repo, params)
-                self.assertEqual(len(versions), 0, versions)
+                with self.assertRaises(HTTPError):
+                    versions = get_versions(self.repo, params)
+                    self.assertEqual(len(versions), 0, versions)
```

Comment:
An HTTP 400 error is returned when getting versions, right? If so, this can be shortened to:

```python
            with self.subTest(params=params):
                with self.assertRaises(HTTPError):
                    get_versions(self.repo, params)
```